babel.ini

```diff
@@ -21,7 +18,6 @@ encoding = utf-8
 
 [jinja2: **/templates/**.jinja]
 encoding = utf-8
-extensions = jinjax.jinjax.JinjaX
```

Comment:
This was added here by @mesemus and I think we need that

Reply:
@mirekys This happened automatically, after using make-translations, I will put it back how it was no issue, but it would be good to know what is happening and also potentially for make-translations to not make such modifications to this file if they are actually unwanted.